src/ontology/cl-edit.owl

```diff
@@ -9566,25 +9581,30 @@ SubClassOf(obo:CL_0000732 obo:CL_0000548)
 
 # Class: obo:CL_0000733 (lymph gland plasmatocyte)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0000733 "A plasmatocyte that derives from the larval lymph gland.")
 AnnotationAssertion(rdfs:label obo:CL_0000733 "lymph gland plasmatocyte")
 SubClassOf(obo:CL_0000733 obo:CL_0000394)
 SubClassOf(obo:CL_0000733 obo:CL_0000735)
 
-# Class: obo:CL_0000734 (embryonic gland plasmatocyte)
+# Class: obo:CL_0000734 (obsolete embryonic gland plasmatocyte)
 
-AnnotationAssertion(rdfs:label obo:CL_0000734 "embryonic gland plasmatocyte")
-SubClassOf(obo:CL_0000734 obo:CL_0000394)
-SubClassOf(obo:CL_0000734 obo:CL_0000736)
+AnnotationAssertion(obo:IAO_0100001 obo:CL_0000734 obo:CL_0011009)
+AnnotationAssertion(rdfs:comment obo:CL_0000734 "Obsoleted as this is the same thing as 'embryonic plasmatocyte'.")
+AnnotationAssertion(rdfs:label obo:CL_0000734 "obsolete embryonic gland plasmatocyte")
+AnnotationAssertion(owl:deprecated obo:CL_0000734 "true"^^xsd:boolean)
 
 # Class: obo:CL_0000735 (lymph gland hemocyte)
```

Comment:
Was this term meant to be obsoleted?

Reply:
Same as above. That term is in use in GO. Its two subclasses are in use as well.